extensions/wikia/TemplateClassification/scripts/TemplateClassificationInView.js

```diff
@@ -14,6 +14,15 @@ define('TemplateClassificationInView', ['jquery', 'mw', 'wikia.nirvana', 'Templa
 		function init() {
 			$typeLabel = $('.template-classification-type-text');
 			templateClassificationModal.init(getType, sendClassifyTemplateRequest);
+
+			$('.template-classification-edit').on('click', function() {
+				tracker.track({
+					trackingMethod: 'both',
+					category: 'template-classification-entry-point',
+					action: 'click',
```

Comment:
I'm thinking of reusing predefined actions to be consistent through whole our system. Let's assume in a meanwhile someone who worked on different feature wants to look into analytics and find info about some events and there is different naming... My opinion is is you mean clicking use predefined var and ensure consistency. Plus it's easier to look through the code for usage of tracker.ACTIONS.CLICK than 'click' in the future.
:)